### PR TITLE
Automation tests for Storybook components

### PR DESCRIPTION
**Pull request by DlgSHi**
# Added to testing scope:

- DateInput,

- DateRange

**Improvements:**

- fixed the name of feature files

### CODE REVIEW

File: cypress/integration/dateInput.feature [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      | fieldHelp                |
+      | Sample text              |
+      | 1234567890               |
+      | áéíóú¿¡üñ                |
+      | !@#$%^*()_+-=~[];:.,?{}  |
+      | ÄÖÜßäöüß                 |
+      | <>                       |
+
+  @positive
+  Scenario Outline: Change DateInput label label
```
Comment (marcinstachowicz) [inline cypress/integration/dateInput.feature]: label label :)

File: cypress/integration/dateInput.feature [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      And I choose date "<day>" via DayPicker
+      And I click dateInput
+    Then the date "<day>" after maxDate is not available
+    Examples:
+      | maxDate     | day              |
+      | 20190509    | Fri May 10, 2019 |
+
+  @positive 
+  Scenario Outline: Change Date Input component label width with slider
+    When I set label to "<label>"
```
Comment (marcinstachowicz) [inline cypress/integration/dateInput.feature]: if we have always "Sample text" as label then step should look like
`I set label to "Sample text"`
and label column can be removed

File: cypress/integration/dateRange.feature [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      | 1234567890               |
+      | áéíóú¿¡üñ                |
+      | !@#$%^*()_+-=~[];:.,?{}  |
+      | ÄÖÜßäöüß                 |
+      | <>                       |
+
+  @positive
+  Scenario Outline: Change Date Range end label
+    When I set endLabel to "<label>"
+    Then end label on preview is "<label>"
```
Comment (marcinstachowicz) [inline cypress/integration/dateRange.feature]: please make `end label` consistant and replace with `endLabel`

File: cypress/support/step-definitions/date-range-steps.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import {
+  labelPreview, errorIcon, dateInput,
+} from '../../locators/date-range/index';
+
+const TEXT_ALIGN = 'text-align';
+const START_LABEL = 'start';
+const END_LABEL = 'end';
+
+Then('start label on preview is {string}', (label) => {
+  labelPreview(1).should('have.text', label);
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/support/step-definitions/date-range-steps.js]: fixed

File: cypress/integration/dateInput.feature (hunk 1 above)
Comment (DlgSHi) [reply cypress/integration/dateInput.feature]: fixed

File: cypress/integration/dateInput.feature (hunk 2 above)
Comment (DlgSHi) [reply cypress/integration/dateInput.feature]: fixed

File: cypress/integration/dateRange.feature (hunk 3 above)
Comment (DlgSHi) [reply cypress/integration/dateRange.feature]: fixed

**Pull request merged**